docs/native_versioning.rst

```diff
@@ -30,3 +30,16 @@ When making schema migrations (for example adding new columns to version tables)
 
 
     sync_trigger(conn, 'article_version')
+
+You also may have a custom option set on the versioning manager called `table_name`, which tells the manager what the versioned table should be called, given a parent table name.  The default is `%s_version`.
+
+If you have this set to something custom, you should instead run
+
+::
+
+    from sqlalchemy_continuum import versioning_manager # or import your custom one, if you have one
+    sync_trigger(conn,
+                 'article_version',
+                 version_table_name_format=versioning_manager.option('table_name'))
```

Comment:
Is there some way to get the versioning manager via context or something? This feels a bit like duplicating configuration. If not, perhaps the best option is to force the user to pass in the entire versioning manager. Then we'd be able to read whatever configuration we need to at any time.

Reply:
I could make sync_trigger take the whole versioning manager - that might actually make sense given that there are other places I will need it too to avoid the hardcoding that's in the native triggers 😬

Since sync_trigger is called manually by user(s) when running alembic migrations, it sort of makes sense that they would have to pass in either configuration or the versioning manager (since it's a different entry-point than using sqlalchemy-continuum in our flask services)

Reply:
Yeah I would opt to pass in the whole versioning_manager. _Unless_ you can somehow use this utility function from within `sync_trigger` function: https://sqlalchemy-continuum.readthedocs.io/en/latest/utilities.html#get-versioning-manager

Reply:
yea, there are no models in scope when doing the native stuff, so that function won't work (was just looking at that too!)

So I will just take the whole manager to make the caller have an easier time with it